src/test/TEST_bench/benchmark_spgemm.jobs

```diff
@@ -26,7 +26,7 @@ mpirun -np 1 ./ij_mm -n 1024 1024 1  -5pt -verify 1 -spgemm_binned 0 -spgemmalg
 mpirun -np 1 ./ij_mm -n 1024 1024 1  -9pt -verify 1 -spgemm_binned 0 -spgemmalg 3 -rep 10 > benchmark_spgemm.out.12
 
 mpirun -np 1 ./ij_mm -n 128 128 128  -7pt -verify 1 -vendor 1                     -rep 10 > benchmark_spgemm.out.13
-mpirun -np 1 ./ij_mm -n 128 128 128 -27pt -verify 1 -vendor 1                     -rep 10 > benchmark_spgemm.out.14
+mpirun -np 1 ./ij_mm -n 80   80  80 -27pt -verify 1 -vendor 1                     -rep 10 > benchmark_spgemm.out.14
```

Comment:
Why did you change the size of the problem?

Reply:
It was because cuda 11 now fails on the older size, out of memory :( . Note that this is with cusparse SpGEMM.